public/docs/_examples/toh-1/ts/demo.css

```diff
@@ -0,0 +1,9 @@
+body, input[text], button {
```

Comment:
Hmmmm it is certainly not doing anything here. Result of copy&paste without looking. Already got it fixed locally, will push with next batch of changes. Not messing with it in the main css file though, as that is used by all the examples.